<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Set_font_color_page</class>
 <widget class="QWidget" name="Set_font_color_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>30</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>30</width>
     <height>200</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
